src/main/java/medicaldoctor/entities/UserInformationGenerator.java

```diff
@@ -0,0 +1,54 @@
+package medicaldoctor.entities;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Random;
+
+public class UserInformationGenerator {
```

Comment:
remember you are working on controllers not the backend services

Reply:
I needed a way to get information such as email. I didn't have it so I made it. I did it so I can test the controller.

Reply:
If you recall in one of our discord convos, controllers should have code that doesn't need to be tested.